src/Reshuffle.ts

```diff
@@ -105,8 +104,22 @@ export default class Reshuffle {
     Object.values(this.registry.connectors).forEach((connector) => connector.start())
 
     // Start the webserver if we have http delegates
-    if (this.registry.common.webserver) {
-      this.httpServer = this.registry.common.webserver.listen(this.port, () => {
+    if (Object.keys(this.httpDelegates).length) {
+      const webserver = this.prepareWebServer()
+      Object.keys(this.httpDelegates)
+        .sort()
+        .reverse() // The sort().reverse() moves all generic routes (/:id) at the end
+        .forEach((path) => {
+          const httpMultiplexer = this.httpDelegates[path]
+          webserver.all(path, httpMultiplexer.handle.bind(httpMultiplexer))
+        })
+      webserver.all('*', (req, res) => {
```

Comment:
This is the default route after everything, it means it wasn't handled before so we return  501